### PR TITLE
App config abstraction class.

### PR DESCRIPTION
**Pull request by pawelzny**
Config object as wrapper around dictionary produced by
file parsers. By default uses YAML parser to load
configuration from file.

It's main goal is to provide separation between file format
and object interface.

### CODE REVIEW

**Comment by magickris93** [comment]
Looks good to me

File: dashboard/config.py [hunk 1]
```diff
@@ -1,6 +1,48 @@
...
+
+    Config object is an wrapper around dictionary produced by YAML parser.
+    It's main goal is to provide separation between config file format
+    and config object interface.
+    Second goal is to provide a way to load more then one config if needed.
+    Can be easily extend to provide more format files
+    by overloading _load() method.
+    """
+    def __init__(self, file_path):
+        if not isinstance(file_path, (str, os.PathLike, bytes)):
```
Comment (kam193) [inline dashboard/config.py]: In another class, we use ``@accepts`` to test type of paramaters, so maybe use this also here? But I'm not sure, if it works fine with methods in classes. 

Info: https://pypi.org/project/accepts/

Comment (kam193) [reply dashboard/config.py]: Ok, in this way, all looks good to me

File: dashboard/config.py [hunk 2]
```diff
@@ -1,6 +1,48 @@
...
+import yaml
+
+
+class Config:
+    """App config abstraction.
+
+    Config object is an wrapper around dictionary produced by YAML parser.
+    It's main goal is to provide separation between config file format
+    and config object interface.
+    Second goal is to provide a way to load more then one config if needed.
```
[the remark this answers is not in the document]
Comment (magickris93) [reply dashboard/config.py]: an wrapper -> a wrapper

**Pull request merged**